joblib/test/test_parallel.py

```diff
@@ -124,27 +125,13 @@ def test_effective_n_jobs():
 
 ###############################################################################
 # Test parallel
-def check_simple_parallel(backend):
-    X = range(5)
-    for n_jobs in (1, 2, -1, -2):
-        assert ([square(x) for x in X] ==
-                Parallel(n_jobs=n_jobs, backend=backend)(
-                    delayed(square)(x) for x in X))
-    for verbose in (2, 11, 100):
-        Parallel(n_jobs=-1, verbose=verbose, backend=backend)(
-            delayed(square)(x) for x in X)
-        Parallel(n_jobs=1, verbose=verbose, backend=backend)(
-            delayed(square)(x) for x in X)
-        Parallel(n_jobs=2, verbose=verbose, pre_dispatch=2,
-                 backend=backend)(
-            delayed(square)(x) for x in X)
-        Parallel(n_jobs=2, verbose=verbose, backend=backend)(
-            delayed(square)(x) for x in X)
-
-
-def test_simple_parallel():
-    for backend in ALL_VALID_BACKENDS:
-        check_simple_parallel(backend)
+
+@parametrize('backend', ALL_VALID_BACKENDS)
+@parametrize('n_jobs', [1, 2, -1, -2])
```

Comment:
You have removed the test on different verbose parameters, I think this is part of the coverage drop.